### PR TITLE
LNK-2982 - ResourceEvaluated and PatientsToQuery changes

### PR DESCRIPTION
**Pull request by nvmLantana**
## Summary by CodeRabbit

- **New Features**
	- Introduced a new enumeration for patient submission statuses, enhancing report management.
	- Added a new method for retrieving submission entries based on specific criteria.
	- Updated models to include a new `ReportType` property for better report categorization.

- **Bug Fixes**
	- Streamlined logic for handling submission entries and report scheduling.

- **Chores**
	- Removed unnecessary properties from models to simplify data handling.
	- Eliminated a listener service related to data acquisition requests, refining service architecture.

### CODE REVIEW

**Comment by MontaltoNick** [comment]
@coderabbitai review

**Comment by nvmLantana** [comment]
@coderabbitai review

File: DotNet/Report/Jobs/GenerateDataAcquisitionRequestsForPatientsToQuery.cs [hunk 1]
```diff
@@ -56,15 +57,12 @@ public async Task Execute(IJobExecutionContext context)
...
-                        ClientId = "Report_DataAcquisitionScheduled"
-                    };
+                var submissionEntries = await _database.SubmissionEntryRepository.FindAsync(e => e.ReportScheduleId == schedule.Id);
+                var itemsToEvaluate = submissionEntries.Where(x => x.Status == PatientSubmissionStatus.NotEvaluated).Select(x => x.PatientId).ToList();
 
-
-                    foreach (string patientId in schedule.PatientsToQuery ?? new List<string>())
+                if (itemsToEvaluate.Any())
+                {
+                    foreach (string patientId in itemsToEvaluate)
```
Comment (arianamihailescu) [inline DotNet/Report/Jobs/GenerateDataAcquisitionRequestsForPatientsToQuery.cs]: Can this potentially create  multiple DA requests for same patient?

Comment (nvmLantana) [reply DotNet/Report/Jobs/GenerateDataAcquisitionRequestsForPatientsToQuery.cs]: Yes, good catch. I have put a distinct() on the patientid list.

**Review by dvargaslantana — APPROVED** [review]

**Pull request merged**